andry_tina_ndvi.py

```diff
@@ -0,0 +1,669 @@
+    
+fig.suptitle("Mean Normalized Difference Vegetation Index (NDVI) from Landsat 8", 
+             x=.52, 
+             y=.95)
+ax.set(title="Harvard Forest (HARV) and San Joaquin Experimental Range (SJER)\nJan 2017 - Dec 2017",
+       xlabel="Month", 
+       ylabel="Mean NDVI")
+ax.legend()
+
+fig.tight_layout()
+
+### DO NOT REMOVE LINES BELOW ###
+final_masked_solution = nb.convert_axes(plt, which_axes="current")
+
+
+# In[11]:
+
+
+# Ignore this cell for the autograding tests
+
+
+# In[12]:
+
+
+# Ignore this cell for the autograding tests
+
+
+# # Question 1 (10 points)
+# 
+# Imagine that you are planning NEON’s upcoming flight season to capture remote sensing data in these locations and want to ensure that you fly the area when the vegetation is the most green.
+# 
+# When would you recommend the flights take place for each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# For HARV, I would recommond flying around summer solstice, and for SJER, I would recommend flying around spring equinox. 
+
+# # Question 2 (10 points)
+# 
+# How could you modify your workflow to look at vegetation changes over time in each site? 
+# 
+# Answer the question in 2-3 sentences in the Markdown cell below.
+
+# For each subsequent scene date, I would calculate the NDVI difference between the new data and the month's prior data. This would allow quantification and visualization of the growth/death of vegetation over the year. 
+
+# # Do not edit this cell! (10 points)
+# 
+# The notebook includes:
+# * additional Markdown cells throughout the notebook to describe: 
+#     * the data that you used - and where it is from
+#     * how data are being processing
+#     * how the code is optimized to run fast and be more concise
+
+# # Do not edit this cell! (20 points)
+# 
+# The notebook will also be checked for overall clean code requirements as specified at the **top** of this notebook. Some of these requirements include (review the top cells for more specifics): 
+# 
+# * Notebook begins at cell [1] and runs on any machine in its entirety.
+# * PEP 8 format is applied throughout (including lengths of comment and code lines).
+# * No additional code or imports in the notebook that is not needed for the workflow.
+# * Notebook is fully reproducible. This means:
+#    * reproducible paths using the os module.
+#    * data downloaded using code in the notebook.
+#    * all imports at top of notebook.
+
+# ## BONUS - Export a  .CSV File to Share (10 points possible)
+# 
+# This is optional - if you export a **.csv** file with the columns specified above: Site, Date and NDVI Value you can get an additional 10 points.
+# 
+# * FULL CREDIT: File exists in csv format and contains the columns specified.
+# We will check your github repo for this file!
+# 
+
+# In[13]:
+
+
+os.chdir(os.path.join(et.io.HOME,
+                      'earth-analytics',
+                      'ea-2021-04-ndvi-automation-c-andry'))
```

Comment:
@c-andry The CI breaks here because the **ea-2021-04-ndvi-automation-c-andry** directory isn't explicitly created in the workbook

Reply:
Right. I lazily kept this in because I didn't know how to get my csv file to automatically end up in the directory that would be pushed back to github